otsdaq-mu2e/FEInterfaces/DTCFrontEndInterface_interface.cc

```diff
@@ -455,75 +543,100 @@ void DTCFrontEndInterface::registerFEMacros(void)
 					&DTCFrontEndInterface::ManualLoopbackSetup),
 				    std::vector<std::string>{"setAsPassthrough", "ROC_Link"},
 					std::vector<std::string>{},
-					1);  // requiredUserPermissions
+					1,    // requiredUserPermissions
+					"*", 
+					"Sets the DTC in loop back mode. This is accomplished by disabling all links except for <b>ROC_Link</b>. "
```

Comment:
Let's standardize "loop back" to one word, "loopback"

Reply:
Loopback (one word) has been standardized across all FE interfaces.